tests/phpunit/elementor/includes/test-frontend.php

```diff
@@ -118,4 +137,35 @@ public function test_get_early_access_google_font_urls() {
 		// Assert
 		$this->assertContains( 'https://fonts.googleapis.com/earlyaccess/opensanshebrew.css', $font_urls );
 	}
+
+	public function test_widget_with_global_scripts() {
+		// Arrange
+		$frontend = new Frontend();
+		$frontend->register_scripts();
+
+		$widget_with_global_scripts = $this->elementor()->widgets_manager->get_widget_types( 'text-editor' );
+
+		// Act
+		$widget_with_global_scripts->enqueue_scripts();
+
+		// Assert
+		$this->assertTrue( wp_script_is( 'elementor-frontend' ) );
+		$this->assertTrue( wp_script_is( 'elementor-frontend-modules' ) );
+		$this->assertTrue( wp_script_is( 'jquery' ) );
+	}
+
+	public function test_widget_without_global_scripts() {
+		// Arrange
+		$frontend = new Frontend();
+		$frontend->register_scripts();
+
+		// Act
+		$widget_without_global_scripts = $this->elementor()->widgets_manager->get_widget_types( 'a-heading' );
```

Comment:
Do we have some test for the config that should be printed with the `frontend` script?